test/integration/targets/filter_urls/runme.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -eux
+
+export ANSIBLE_ROLES_PATH=../
+
+ansible-playbook -i ../../inventory runme.yml "$@"
+
+source virtualenv.sh
+
+# This has to be done outside of requirements.txt but is necessary for
+# installing Jinja 2.6
+pip install 'setuptools<45'
+
+pip install -U -r requirements.txt
+
+ansible-playbook -i ../../inventory jinja26.yml -v "$@"
```

Comment:
No need for inventory for this test since filters only run on the controller:

```suggestion
ansible-playbook jinja26.yml -v "$@"
```